config/ftbquests/quests/chapters/immersive_engineering.snbt

```diff
@@ -13,7 +13,12 @@
 			x: -1.5d
 			y: -6.0d
 			shape: "gear"
-			description: ["Immersive Engineering is a versatile tech mod with a focus on large multiblock structures. You can find more information about anything in the mod in the Engineer's Handbook."]
+			description: [
+				"Immersive Engineering is a versatile tech mod with a focus on large multiblock structures. You can find more information about anything in the mod in the Engineer's Handbook."
+				""
+				"Edited for E8 purposes by ~~Kanzaji"

```

Comment:
purposes doesn't really fit into this sentence.

Reply:
👍 Fixing it :D
Tbh I think I will just remove this note, as the note about port is there so.